glusterd2/commands/volumes/volume-create.go

```diff
@@ -17,6 +18,12 @@ import (
 )
 
 func validateVolCreateReq(req *api.VolCreateReq) error {
+	reg := regexp.MustCompile("^[a-zA-Z0-9_-]+$")
```

Comment:
Move this to global scope, outside this function

Reply:
done